test.sh

```diff
@@ -8,6 +8,7 @@ declare -A testCases=(
     [complete_values]="--values test-values.yaml"
     [overrides]="--set coordinatorNameOverride=coordinator-overridden,workerNameOverride=worker-overridden,nameOverride=overridden"
     [access_control_properties_values]="--values test-access-control-properties-values.yaml"
+    [exchange_manager_values]="--values test_access_exchange_manager_values.yaml
```

Comment:
Missing terminating double quote:
```suggestion
    [exchange_manager_values]="--values test_access_exchange_manager_values.yaml"
```